thinc/layers/__init__.py

```diff
@@ -22,8 +23,14 @@
 from .noop import noop
 from .recurrent import recurrent
 from .uniqued import uniqued
+from .siamese import siamese
```

Comment:
```suggestion
from .siamese import Siamese
```

Reply:
I want that layer to be `siamese`, as it's a weightless combinator.